Sources/StreamChat/WebSocketClient/EventMiddlewares/EventConverterMiddleware.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The middleware responsible for converting event DTOs to events.
+struct EventConverterMiddleware: EventMiddleware {
```

Comment:
It is probably a better and more clear name 👍 
```suggestion
/// The middleware responsible for converting event DTOs to events.
struct EventDTOConverterMiddleware: EventMiddleware {
```

Reply:
✅ Renamed